client/src/components/NumOfRandomJobsInput/styles.module.scss

```diff
@@ -0,0 +1,3 @@
+:local(.numOfRandomJobs):global(.MuiFormControl-root) {
```

Comment:
Use a new line here

Reply:
This could be misleading as in CSS it is a common practise to write selectors joined with comma on separate lines. But that has a different meaning, so it is better to leave it as it is

Reply:
> A comma-separated list of selectors represents the union of all elements selected by each of the individual selectors in the list. For example, in CSS when several selectors share the same declarations, they may be grouped into a comma-separated list.